cross/libicu/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = icu
+PKG_VERS = 64.2
+PKG_EXT = tgz
+PKG_DIST_NAME = icu4c-64_2-src.$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/unicode-org/icu/releases/download/release-64-2/
+PKG_DIR = icu/source
+
+DEPENDS = 
+
+HOMEPAGE = http://icu-project.org/
+COMMENT  = ICU is a mature, widely used set of C/C++ libraries providing Unicode and Globalization support for software applications.
+LICENSE  = GPLv2
+
+GNU_CONFIGURE = 1
+
+PRE_CONFIGURE_TARGET = pre_configure_icu
+
+SHARED_WORK_DIR = $(shell pwd)/work
+ICU_WORK_DIR = $(SHARED_WORK_DIR)/icu
+
+SHARED_CONFIGURE_ARGS = --disable-tests --disable-samples --disable-extras
+CONFIGURE_ARGS = $(SHARED_CONFIGURE_ARGS) --with-cross-build=$(ICU_WORK_DIR)
+
+UNSUPPORTED_ARCHS = 88f6281 
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: pre_configure_icu
+pre_configure_icu:
+	if [ ! -d "$(ICU_WORK_DIR)" ]; then \
+		mkdir -p $(ICU_WORK_DIR); \
+		cd $(ICU_WORK_DIR) \
+			&& RANLIB= LDFLAGS= CXXFLAGS= CXX= CPPFLAGS= CPP= CFLAGS= CC= AR= ARFLAGS= $(WORK_DIR)/$(PKG_DIR)/configure $(SHARED_CONFIGURE_ARGS) \
+			&& make; \
+	fi;
```

Comment:
I just guess icu requires a native compilation first to be able to cross-compile in a second phase. If so, please create a `native/icu` module and refers to it in a similar way as `cross/protobuf` and `native/protobuf`.

Reply:
Thanks for the tip, I had struggled with it !

- [ ] Create native/icu module

Reply:
(I might as well try removing the requirement for icu altogether... It was actually required to build the wheels)

Reply:
OK, turns out that ICU dependency was no longer required.
At least I learnt how it could be done :)
Thanks anyway for the help since, indeed, the native option was way simplier here !